compiler/src/dotty/tools/dotc/Driver.scala

```diff
@@ -64,19 +64,23 @@ class Driver {
 
   protected def sourcesRequired: Boolean = true
 
-  def setup(args: Array[String], rootCtx: Context): (List[AbstractFile], Context) = {
+  protected def command: CompilerCommand = ScalacCommand
+
+  def setup(args: Array[String], rootCtx: Context): (Option[List[AbstractFile]], Context) = {
```

Comment:
An Option of a List is weird, the documentation of this method should explain clearly how None differs from Some(Nil) here.

Reply:
Will update documentation